federatedscope/vertical/dataloader/dataloader.py

```diff
@@ -39,15 +68,17 @@ def load_vertical_data(config=None, generate=False):
         # For Client #1
         data[1] = dict()
         data[1]['train'] = {
-            'x': x[:train_num, :config.vertical.dims[0]],
-            'y': y[:train_num]
+            'x': x[:train_num, :config.caesar_vertical.dims[0]]
         }
         data[1]['val'] = None
         data[1]['test'] = test_data
 
         # For Client #2
```

Comment:
So we assume that client_2 owns the labels? maybe we can add some annotations here.

Reply:
get it